style.css

```diff
@@ -0,0 +1,156 @@
+* {
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+a {
+  cursor: pointer;
+}
+
+.hide {
+  display: none;
+}
+
+.social-media {
+  margin: 5% 1%;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
```

Comment:
- **[OPTIONAL]** I recommend you use `align-items: center;` instead of `align-items: baseline;` to be similar to the Figma template and correct the color of the Twitter icon 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/227409218-9e32c6e2-0d8f-4d74-a8c4-544f925005e9.png)